src/pip/_internal/commands/debug.py

```diff
@@ -7,9 +7,13 @@
 import logging
 import os
 import sys
+from types import ModuleType
 
+import pip._vendor
+import pkg_resources
```

Comment:
This needs to be imported from `from pip._vendor import pkg_resources`

Reply:
done